tests/contract/client/tests/test_merino.py

```diff
@@ -114,12 +118,22 @@ def merino_step(step: Step) -> None:
             # instance for checking the field types and comparing a dict
             # representation of the model instance with the expected response
             # content for this step in the test scenario.
-            assert_200_response(
-                step_content=step.response.content,
-                merino_content=ResponseContent(**response.json()),
-                fetch_kinto_icon_url=fetch_kinto_icon_url,
-            )
-            return
+
+            if step.request.path == "/__version__":
```

Comment:
I suggest to change this condition to use the expected response type instead of the request path

```suggestion
            if type(step.response.content) == VersionResponseContent:
```